eth/api_backend.go

```diff
@@ -21,7 +21,7 @@ import (
 	"errors"
 	"math/big"
 
-	"github.com/spruce-solutions/go-quai"
+	ethereum "github.com/spruce-solutions/go-quai"
```

Comment:
Why is this adding Ethereum again?

Reply:
this is happening again. Something changed after one our previous PR's. Even your PR has this. Need to take a closer look